packages/manager/modules/log-to-customer/src/list/controller.js

```diff
@@ -0,0 +1,164 @@
+import moment from 'moment';
+import { RETENTION } from '../constants';
+
+export default class LogToCustomerListCtrl {
+  /* @ngInject */
+  constructor($state, coreURLBuilder, LogToCustomerService, $translate, $q) {
+    this.$state = $state;
+    this.coreURLBuilder = coreURLBuilder;
+    this.LogToCustomerService = LogToCustomerService;
+    this.$translate = $translate;
+    this.$q = $q;
+  }
+
+  $onInit() {
+    this.loading = false;
+    this.accountList = [];
+    this.streams = [];
+    this.streamSubscriptions = {};
+    this.LDPHomePageLink = null;
+    this.selectedAccount = null;
+
+    this.LogToCustomerService.getIcebergLogAccounts().then((data) => {
+      this.accountList = data.map((item) => {
+        return {
+          ...item,
+          formattedName: item.displayName || item.serviceName,
+        };
+      });
+
+      this.selectedAccount =
+        this.accountList.length > 0 ? this.accountList[0] : null;
+      this.getStreamData();
+    });
+  }
+
+  getStreamData() {
+    this.$q
+      .all([this.getSubscribedStreams(), this.getLogAccountStreams()])
+      .then(() => {
+        this.loading = false;
+      });
```

Comment:
You need to catch potential errors
```suggestion
       .catch((error) => {
           this.errorMessage = error?.data?.message;
       })
      .finally(() => {
        this.loading = false;
      });
```